seedDatabase/seed.js

```diff
@@ -0,0 +1,209 @@
+const uuidv4 = require('uuid');
+const faker = require('faker');
+const LoremIpsum = require('lorem-ipsum').LoremIpsum;
+const pool = require('./db_PostgreSQL.js');
+const fs = require('graceful-fs');
+const videosArray = require('./videosArray.js');
+const csvWriter = require('csv-write-stream');
+const TaskTimer = require('tasktimer').TaskTimer;
+
+const lorem = new LoremIpsum({
+  sentencesPerParagraph: {
+    max: 8,
+    min: 4
+  },
+  wordsPerSentence: {
+    max: 16,
+    min: 4
+  }
+});
+
+const courseFilePath = `${__dirname}/copy_data/courses6.csv`;
+const sectionFilePath = `${__dirname}/copy_data/sections6.csv`;
+const elementFilePath = `${__dirname}/copy_data/elements6.csv`;
+
+const numOfSubs = function(arr) {
+  return arr[Math.floor(Math.random() * arr.length)];
+};
+
+const randomKindWithProb = function() {
+  const kinds = ['lecture', 'lecture', 'lecture', 'lecture', 'lecture', 'lecture', 'lecture', 'exercise', 'exercise', 'quiz', 'article', 'article', 'article', 'quiz', 'lecture', 'lecture', 'exercise'];
+  return kinds[Math.floor(Math.random() * kinds.length)];
+};
+
+const writeRecord = async(obj, filePath) => {
+  const writer = csvWriter( {sendHeaders: false} );
+  writer.pipe(fs.createWriteStream(filePath, {flags: 'a'}));
+  writer.write(obj);
+  writer.end();
+};
+
+const seedElement = async(courseId, sectionId) => {
+  let records = [];
+
+  let length = 0;
+  let lectureCount = 0;
+  let quizCount = 0;
+  let exerciseCount = 0;
+  let articleCount = 0;
+
+  let n = numOfSubs([1, 2, 3, 4]);
+
+  for (let i = 0; i < n; i++) {
+    let obj = {};
+    obj['id'] = uuidv4();
+    obj['course_id'] = courseId;
+    obj['section_id'] = sectionId;
```

Comment:
just a note that you probably already know, but you can use dot notation to avoid brackets and a quotes
e.g. `obj.id = uuidv4();`